### ISSUE TITLE
[🔷 Feature request ]: Aliases for feedForward & backpropagate

### ISSUE THREAD

**Issue by matiasvlevi** [matiasvlevi/Dann#26]
### Feature
We could have aliases for the `feedForward` & `backpropagate` methods of the `Dann` class.

### Type
- [x] Dann
- [ ] Matrix
- [ ] Layer
- [ ] Activation functions
- [ ] Loss functions
- [ ] Pool functions
- [ ] Datasets
- [ ] Documentation
- [ ] tests & examples
- [ ] Other 

### Description
These aliases would be to make method names a little more uniform when we add in the `Rann` class for RNNs as `feedForward` & `backpropagate` would not be the most accurate terms for RNNs. See the issue on RNNs [here](https://github.com/matiasvlevi/Dann/issues/25).

#### Feedforward
Is located in `src/classes/dann/methods/feedForward.js`

#### backpropagate
Is located in `src/classes/dann/methods/backpropagate.js`

### Examples
These would be the aliases
`Dann.prototype.backpropagate` to `Dann.prototype.train`
`Dann.prototype.feedForward` to `Dann.prototype.feed`

### Note
It is important to note that we do not want to remove `backpropagate` & `feedForward` names, I think having machine learning terms for methods helps to get a grasp at what the neural network is doing since you can look up the terms.

**Comment by SharkAce** [comment]
I'm on it man

**Comment by matiasvlevi** [comment]
Just merged your fork! Thanks!
